multiplexed-display.ino

```diff
@@ -78,6 +78,72 @@ void printContents() {
   }
 }
 
+void programmEEPROM() {
+	byte digits[] = { 0x7e, 0x30, 0x6d, 0x79, 0x33, 0x5b, 0x5f, 0x70, 0x7f, 0x7b };
+	byte minusSign = 0x01;
+
+	int progressStep = 16;
+	Serial.println("Programming EEPROM:");
+
+	// Go over all inputs
+	for (byte i = 0; i < 256; i++) {
+		if (i % progressStep == 0) {
+			Serial.print((i / 256) * 100);
+      Serial.println("%");
+		}
+
+		// Non two's complement part
+		{
+			int d2 = (i / 100) % 10; // 2nd 7segment display (hundreds)
+			int d3 = (i / 10) % 10; // 3rd 7segment display (tenths)
+			writeEEPROM(i + 512 + 256, 0); // clear first 7segment display
+			writeEEPROM(i + 512, (d2 == 0) ? 0 : digits[d2]); // write the digit or clear if 0
+			writeEEPROM(i + 256, (d2 == 0 && d3 == 0) ? 0 : digits[d3]); // write the digit or clear if d3 and d2 are 0
+			writeEEPROM(i, digits[i % 10]); // write the digit even 0
+		}
```

Comment:
It's easier to show powers of two, when using hex notation, like this:
      writeEEPROM(i + **0x300**, blank); // clear first 7segment display
      writeEEPROM(i + **0x200**, (d2 == 0) ? blank : digits[d2]); // write the digit or clear if 0
      writeEEPROM(i + **0x100**, (d2 == 0 && d3 == 0) ? blank : digits[d3]); // write the digit or clear if d3 and d2 are 0
      writeEEPROM(i + **0x000**, digits[i % 10]); // write the digit even 0